From 5463dd647154729fd0c5a93e3e170546b90cad9d Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <contact@chrisaw.io>
Date: Mon, 14 Sep 2026 23:15:17 +0100
Subject: [PATCH 2/2] Use GrapheneOS Messaging module name in GSI system image

GrapheneOS replaced the AOSP messaging app with its own fork at
external/Messaging, where the soong module is named Messaging, so the
old AOSP module name leaves aosp_system_image depending on an undefined
module and soong bootstrap fails.
---
 target/product/gsi/Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/product/gsi/Android.bp b/target/product/gsi/Android.bp
index 8a6e692a1e..1dea323ab5 100644
--- a/target/product/gsi/Android.bp
+++ b/target/product/gsi/Android.bp
@@ -124,7 +124,7 @@ android_filesystem_defaults {
         ///////////////////////////////////////////
         "Dialer",
         "LatinIME",
-        "messaging",
+        "Messaging",
         "TrichromeChrome",
         "frameworks-base-overlays",
         "preinstalled-packages-base-product.xml",
-- 
2.55.0

